### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by DavidMinarsch**
## Release summary

Version number: 0.1.8

## Release details

This release:

- fixes miscellaneous bugs
- improves the CI
- adds results from 102 simulations

## Checklist

_Put an `x` in the boxes that apply._

- [x] I have read the [CONTRIBUTING](../master/CONTRIBUTING.rst) doc
- [x] I am making a pull request against the `master` branch (left side), from `develop`
- [x] Lint and unit tests pass locally
- [x] I built the documentation and updated it with the latest changes
- [x] I've added an item in `HISTORY.rst` for this release
- [x] I bumped the version number in the `tac/__version__.py` file.
- [ ] I bumped the version number in every Docker image of the repo and published it. Also, I built and published them with tag `latest` 

## Further comments

-

### CODE REVIEW

File: sandbox/sim.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+python tac/gui/dashboards/leaderboard.py --datadir data/shared/EX_2
```
Comment (DavidMinarsch) [inline sandbox/sim.md:1]: Needs removing

File: tac/agents/controller/agent.py [hunk 2]
```diff
@@ -178,7 +178,7 @@ def start(self) -> None:
         """
         try:
             super().start()
-            logger.debug("[{}]: Starting myself...".format(self.name))
+            logger.debug("[{}]: Stopping myself...".format(self.name))
```
Comment (DavidMinarsch) [inline tac/agents/controller/agent.py:181]: @marcofavorito why did you update that..seems wrong

File: tac/agents/controller/agent.py [hunk 3]
```diff
@@ -200,7 +200,6 @@ def stop(self) -> None:
         if self.game_handler.game_phase == GamePhase.GAME or self.game_handler.game_phase == GamePhase.GAME_SETUP:
             self.game_handler.notify_competition_cancelled()
         super().stop()
-        self.teardown()
```
Comment (DavidMinarsch) [inline tac/agents/controller/agent.py:203]: So where is teardown called? In super?

File: tac/agents/participant/v1/agent.py [hunk 4]
```diff
@@ -145,6 +145,7 @@ def start(self, rejoin: bool = False) -> None:
             self.oef_handler.rejoin = rejoin
             super().start()
             self.oef_handler.rejoin = False
```
Comment (DavidMinarsch) [inline tac/agents/participant/v1/agent.py:147]: hmm ...not sure why we are calling stop in start?

File: sandbox/sim.md (hunk 1 above)
Comment (marcofavorito) [reply sandbox/sim.md:1]: OK

File: tac/agents/controller/agent.py (hunk 2 above)
Comment (marcofavorito) [reply tac/agents/controller/agent.py:181]: That line is executed when the agent exits from the main loop (in `super().start()`).

File: tac/agents/controller/agent.py (hunk 3 above)
Comment (marcofavorito) [reply tac/agents/controller/agent.py:203]: yes. `super().stop()` calls `self.teardown()`

File: tac/agents/participant/v1/agent.py (hunk 4 above)
Comment (marcofavorito) [reply tac/agents/participant/v1/agent.py:147]: I agree it shouldn't be like that. However, somehow the main loop was exited without calling `stop()` but setting the liveness manually, hence bypassing the call to `self.stop()` and hence also the call to `self.teardown()`

**Pull request merged**